src/SegmentedArc.js

```diff
@@ -221,7 +215,9 @@ SegmentedArc.propTypes = {
   rangesTextStyle: PropTypes.object,
   capInnerColor: PropTypes.string,
   capOuterColor: PropTypes.string,
-  alignRangesWithSegments: PropTypes.bool
+  alignRangesWithSegments: PropTypes.bool,
+  dataErrorComponent: PropTypes.oneOfType([PropTypes.func, PropTypes.elementType]),
```

Comment:
Same here, after latest changes we don't need func type

Reply:
same here, if we use PropTypes.element this should take care of function being passed